FinniversKit/FinniversKit.xcodeproj/project.pbxproj

```diff
@@ -2948,14 +2954,6 @@
 			path = PromotionView;
 			sourceTree = "<group>";
 		};
-		DA5A858A24617DE200AC0DFE /* SwiftUI */ = {
-			isa = PBXGroup;
-			children = (
-				DA5A858B24617DF000AC0DFE /* ImageExtensions.swift */,
-			);
-			path = SwiftUI;
-			sourceTree = "<group>";
-		};
```

Comment:
Is this folder already removed from the project, or was it removed unintentionally?

Reply:
Sorry, nevermind! Just noticed the file `ImageExtensions.swift` was moved in this PR 😄